src/Inputam.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
```

Comment:
Where this file is used ??
Please name this file correctly.
Have you not reviewed your PR as we discussed in last call ?

Reply:
This is unused file . from next time i will be remember that these file not push on github.